src/urlGenerator.ts

```diff
@@ -0,0 +1,12 @@
+import { resolve } from 'url';
+
+class UrlGenerator {
```

Comment:
it does not make sense to have a class for this util function. Maybe call it `url-utils`?

Reply:
Agree